basiclti/basiclti-tool/src/webapp/vm/lti_tool_edit.vm

```diff
@@ -0,0 +1,393 @@
+<div class="portletBody">
+    <script type="text/javascript" src="/library/js/headscripts.js"></script>
+    <script>
+        includeLatestJQuery('lti_tool_edit.vm');
+        includeWebjarLibrary('fontawesome-iconpicker');
+    </script>
+    <h3>$tlang.getString("tool.put")</h3>
+
+    #if ($alertMessage)<div class="sak-banner-error">$tlang.getString("gen.alert") $formattedText.escapeHtml($alertMessage)</div>#end
+
+    <form action="#toolForm("")" method="post" name="customizeForm" >
+        #if ($isEdit && $isAdmin && $autoStartUrl )
+            <div style="float:right" id="lti-advantage-div">
```

Comment:
avoid inline style if possible?

Reply:
I don't think so - This is not a general thing - it is just one button that needs to float right.  Any other approach would be harder to understand for future developers.